usaspending_api/submissions/management/commands/load_submission.py

```diff
@@ -289,6 +289,15 @@ def handle(self, *args, **options):
             award = load_data_into_model(Award(), row, field_map=award_field_map, value_map=award_value_map, as_dict=True)
             award, created = Award.objects.get_or_create(**award)
 
+            fad_value_map = {
+                "award": award,
+                "submission": submission_attributes
+
+            }
+
+            financial_assistance_data = load_data_into_model(FinancialAssistanceAward(), row, value_map=fad_value_map, as_dict=True)
+            fad, created = FinancialAssistanceAward.objects.get_or_create(**financial_assistance_data)
```

Comment:
🤔 Hmmm so, it could definitely exist from the usaspending loader, because that data is updated more frequently and on an ongoing/rolling basis, whereas the broker submissions are quarterly. Not sure if it will appear in a previous broker submission but since both loaders use the same tables, I think my comment still stands.